src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,13 +672,13 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         Error(diagnostics, ErrorCode.ERR_AnonymousTypeDuplicatePropertyName, argumentSyntax.NameColon.Name);
```

Comment:
@AlekseyTs 
If you type `(int a, int, int a)`, you will get:
1. An error on the second member (because it needs a name)
2. An error on the third member (because it needs a unique name)

The user will have to fix those two distinct errors, so I think it is good to report them together.